README.md

```diff
@@ -1,2 +1,51 @@
-# the-next-web-clone
-The Next Web Clone
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The Next Web Clone
+
+> A clone of The Next Web website practicing the principles of responsiveness using media queries
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] The screenshot of your web app is not showing. The path to the file may be incorrect. Kindly check.